aws/data_source_aws_ec2_coip_pool.go

```diff
@@ -34,6 +34,12 @@ func dataSourceAwsEc2CoipPool() *schema.Resource {
 				Computed: true,
 			},
 
+			"pool_arn": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
Remove `Optional` attribute. It would be applicable if the attribute could be specified by the user, but that is not the case here.

Reply:
Thanks @ewbankkit : Have removed the optional attribute and changed the name from pool_arn to arn.